managers/tests/test_unit_templatetags.py

```diff
@@ -0,0 +1,104 @@
+import datetime
```

Comment:
After moving template tags as custom manager methods please remove those tests too. Also I don't think there should be extra tests for custom manager methods, as those will be just some simple querysets.

Reply:
Done